apps/easypid/src/features/activity/FunkeActivityScreen.tsx

```diff
@@ -69,7 +69,8 @@ export function FunkeActivityScreen() {
                       subtitle={activity.entityName ?? activity.entityHost}
                       date={new Date(activity.date)}
                       type={activity.type}
-                      credentialId={activity.credentialId}
+                      // FIXME: Handle multiple credentials received in one request
```

Comment:
we could also store them as separate activities if that's simpler?

Reply:
It works fine, there is just no support for receiving multiple credentials in 1 request atm so will do this later.